Packs/Azure/Integrations/Azure/Azure.yml

```diff
@@ -536,6 +536,309 @@ script:
+      name: append
+      defaultValue: False
+    description: Sets the tags for the specified Blob. The command replace the entire tags of the Blob and can be used to remove tags.
+    name: azure-storage-container-blob-tag-set
+
+  - arguments:
+    - description: The name of the Blob Container.
+      name: container_name
+      required: true
+    - description: The name of the blob.
+      name: blob_name
+      required: true
+    - description: "The subscription ID."
+      name: subscription_id
+      required: true
+    - description: "The resource group name."
+      name: resource_group_name
+      required: true
+    - description: The storage account name.
+      name: account_name
+      required: true
+    description: Retrieve Blob properties.
+    name: azure-storage-container-blob-property-get
+    outputs:
+    - contextPath: Azure.StorageContainer.Container.Blob.Property.last_modified
+      description: Last modified time of the blob.
+      type: Date
+    - contextPath: Azure.StorageContainer.Container.Blob.Property.etag
+      description: The entity tag for the blob.
+      type: String
+    - contextPath: Azure.StorageContainer.Container.Blob.Property.lease_status
+      description: The lease status of the blob.
+      type: String
+    - contextPath: Azure.StorageContainer.Container.Blob.Property.lease_state
+      description: The lease state of the blob.
+      type: String
+    - contextPath: Azure.StorageContainer.Container.Blob.Property.blob_type
+      description: The blob type.
+      type: String
+    - contextPath: Azure.StorageContainer.Container.Blob.Property.content_length
+      description: The size of the blob in bytes.
+      type: Number
+    - contextPath: Azure.StorageContainer.Container.Blob.Property.content_type
+      description: The content type specified for the blob. If no content type was specified, the default content type is application/octet-stream.
+      type: String
+    - contextPath: Azure.StorageContainer.Container.Blob.Property.content-md5
+      description: The MD5 hash of the blob content.
+      type: String
+    - contextPath: Azure.StorageContainer.Container.Blob.Property.creation_time
+      description: The date at which the blob was created.
+      type: Date
+    - contextPath: Azure.StorageContainer.Container.Blob.name
+      description: Blob name.
+      type: String
+
+  - arguments:
+    - description: The name of the Blob Container.
+      name: container_name
+      required: true
+    - description: The name of the blob.
+      name: blob_name
+      required: true
+    - description: "The subscription ID."
+      name: subscription_id
+      required: true
+    - description: "The resource group name."
+      name: resource_group_name
+      required: true
+    - description: The storage account name.
+      name: account_name
+      required: true
+    - description: Blob content type. Indicates the media type of the blob.
+      name: content_type
```

Comment:
Is there a close list of types that we can use for creating a dropdown list of types?

Reply:
There is no close list for this